forms/viewsets/form.py

```diff
@@ -90,6 +90,9 @@ class AttachmentViewSet(viewsets.ModelViewSet):
 
     def get_queryset(self):
         qs = super().get_queryset()
+        user = self.request.user
+        if user.has_perm("forms.view_attachment"):
```

Comment:
Should the permission name be view_attachments. Since this is returning all the attatchments if a read this correctly?

Reply:
Very good point but this is djnago created permission name so it could be stupid to create another permission called view_attachments